movieDetail.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Movie Box</title>
+    <link rel="stylesheet" href="./assests/styleSheets/movieDetail.css" />
+  </head>
+  <body>
+    <main>
+      <div class="details-container">
```

Comment:
you can remove this `div` and add the class to `main`.

Reply:
they both have their own class :(

Reply:
this separation is for mobile

Reply:
they both have their own design and without it I can not handle the UI design :(